pgjdbc/src/test/java/org/postgresql/test/jdbc2/TimestampTest.java

```diff
@@ -577,30 +578,54 @@ private void timestampTestWOTZ() throws SQLException {
       assertNotNull(t);
       assertTrue(t.equals(TS1WOTZ));
 
+      tString = rs.getString(1);
+      assertNotNull(tString);
+      assertTrue(tString.equals(TS1WOTZ_PGFORMAT));
```

Comment:
+1

While the surrounding code uses `assertTrue(...equals)` pattern,  please use `assertEquals(expected, actual)` so expected and actual values get printed in the failure case. See more in https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md#test